iset.mm

```diff
@@ -60969,6 +61036,73 @@ the cancellation property (fifth hypothesis), show that the relation
       JUHAASTUAUBUC $.
   $}
 
+  ${
+    $d f x y z w A $.  $d f B $.  $d x y z w F $.
+    $( A version of ~ fnex for finite sets that does not require Replacement.
+       (Contributed by Mario Carneiro, 16-Nov-2014.)  (Revised by Mario
+       Carneiro, 24-Jun-2015.) $)
+    fnfi $p |- ( ( F Fn A /\ A e. Fin ) -> F e. Fin ) $=
+      ( vw vy vz cfn wcel wa cres cv c0 csn cun reseq2 eleq1d syl2anc cvv simpr
+      wceq cdm wfn fnresdm adantr res0 0fin eqeltri a1i wss resundi cfv simp-4l
+      cdif simplrr eldifad fnressn uneq2d wn elexd funfvex funfni opexg eldifbd
+      cop cin opeldmg dmres eleq2i syl6ib elin simplbi syl6 mtod unsnfi syl3anc
+      wi eqeltrd syl5eqel ex findcard2sd eqeltrrd ) BAUAZAFGZHZBAIZBFWAWDBSWBAB
+      UBUCWCBCJZIZFGBKIZFGZBDJZIZFGZBWIEJZLZMZIZFGZWDFGCDEAWEKSWFWGFWEKBNOWEWIS
+      WFWJFWEWIBNOWEWNSWFWOFWEWNBNOWEASWFWDFWEABNOWHWCWGKFBUDUEUFUGWCWIFGZHZWIA
+      UHZWLAWIULZGZHZHZWKWPXCWKHZWOWJBWMIZMZFBWIWMUIXDXFWJWLWLBUJZVCZLZMZFXDXEX
+      IWJXDWAWLAGZXEXISWAWBWQXBWKUKZXDWLAWIWRWSXAWKUMZUNZAWLBUOPUPXDWKXHQGZXHWJ
+      GZUQXJFGXCWKRXDWLQGXGQGZXOXDWLWTXMURXDWAXKXQXLXNXQAWLBWLBUSUTPZWLXGQQVAPX
+      DXPWLWIGZXDWLAWIXMVBXDXPWLWIBTZVDZGZXSXDXPWLWJTZGZYBXDXKXQXPYDVOXNXRWLXGW
+      JAQVEPYCYAWLBWIVFVGVHYBXSWLXTGWLWIXTVIVJVKVLWJXHQVMVNVPVQVRWAWBRVSVT $.
+  $}
+
+  $( The domain of a finite function is finite.  (Contributed by Jim Kingdon,
+     5-Feb-2022.) $)
+  fundmfi $p |- ( ( A e. Fin /\ Fun A ) -> dom A e. Fin ) $=
+    ( cfn wcel wfun cdm cen wbr fundmeng enfii syldan ) ABCADAEZAFGKBCABHKAIJ
+    $.
+
+  $( A function (set) is finite if and only if its domain is finite.
```

Comment:
why "(set)" ?

Reply:
... because A is also a set. But maybe this can be removed.

Reply:
I would remove it, since the antecedent is simply "Fun F".

Reply:
I cannot find fundmfi in set.mm anymore. Maybe it was removed in favour of ~fundmfibi.

Reply:
Although it is true that this theorem doesn't apply for a function which is a proper class (because both sides of the biconditional imply the function is a set), sounds like we are agreed that there isn't any particular need to emphasize that and we can drop "(set)" from the comment. (Plus if we did want to emphasize it, I think we'd want a more explicit wording).

As for `fundmfi`, it is added in this pull request and corresponds to http://us.metamath.org/mpeuni/dmfi.html in set.mm (as noted in the mmil.html entry

```
<TR>
  <TD>dmfi</TD>
  <TD>~ fundmfi</TD>
</TR>
```

which is part of this pull request).

Reply:
> there isn't any particular need to emphasize that and we can drop "(set)" from the comment. (Plus if we did want to emphasize it, I think we'd want a more explicit wording).

I agree with both remarks.